### DISCUSSION TITLE
Adding oggenc2?

### DISCUSSION THREAD

**Discussion by aposorio** [enzo1982/freac#363] [Help & Support] [answered]
I'm attempting to add the oggenc2 encoder from https://www.rarewares.org/ogg-oggenc.php (this version in particular: https://www.rarewares.org/ogg-oggenc.php#oggenc-aotuv) but have been unable to do so.

Any guidance would be much appreciated.

Thanks in advance.

**Comment by enzo1982** [comment, marked as answer]
fre:ac uses XML configuration scripts to integrate external codecs and there was no script for OggEnc2 yet.

The latest [continuous build](https://github.com/enzo1982/freac/releases/tag/continuous) adds one, so with that version all you need to do is put the oggenc2.exe executable in the codecs\cmdline folder inside the fre:ac application folder. The encoder will then appear as _OggEnc2 Ogg Vorbis Encoder_ in fre:ac.

By the way, the Ogg Vorbis encoder that comes with fre:ac already uses the aoTuV patches, so if it's that which you were looking for, you can just use the integrated encoder.

**Reply by aposorio** [reply]
Thanks for your help. I was able to add to add Oggenc2.88 using the continuous build. 

I appreciate your time and assistance.
